### DISCUSSION TITLE
Proposed Documentation Improvement

### DISCUSSION THREAD

**Discussion by diego-drese** [laravel/nova-issues#6464] [Q&A] [answered]
Hello, 

I'd like to propose an enhancement to the documentation regarding custom notifications in Laravel Nova, specifically to clarify the use of the `toNova()` method to ensure additional parameters like `iconClass` are correctly stored in the database.

### Current Documentation Issue
In the current documentation for custom notifications (https://nova.laravel.com/docs/customization/notifications.html), there is no clear mention that the `toNova()` method must be called to ensure parameters like `iconClass` are saved properly. This can lead to confusion and developers might miss the step, causing the `iconClass` parameter not to be stored in the database.

### Proposed Documentation Improvement
Update the example in the documentation to clearly demonstrate the need to call the `toNova()` method. Here is the proposed update:
**Current Example:**
   ```php
   public function toNova()
   {
       return (new NovaNotification)
           ->message('Your report is ready to download.')
           ->action('Download', URL::remote('https://example.com/report.pdf'))
           ->icon('download')
           ->type('info');
   }
   ```
   **Propose exemple:**
  ```php
   public function toNova()
{
    return (new NovaNotification)
        ->message('Your report is ready to download.')
        ->action('Download', URL::remote('https://example.com/report.pdf'))
        ->icon('download')
        ->type('info')
        ->toNova(); // Ensure to call toNova() to correctly store additional parameters like iconClass
}
```

**Comment by anditsung** [comment, marked as answer]
you can try to create a PR here https://github.com/laravel/nova-docs
